### PR TITLE
SolarEdge energy dashboard configuration example

### PR DESCRIPTION
**Pull request by purcell-lab**
## Proposed change

This change documents the sensors to be utilised for the SolarEdge integration to populate the Energy Dashboard, which is natively supported as of core-2021.11.

This addresses a common question as to how to configure the Energy Dashboard with this integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR documents the changes in issue: https://github.com/home-assistant/core/issues/58735#event-5545327262

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by purcell-lab** [comment]
Should be proposed for `next` branch but I couldn't work out how to point against that branch.

File: source/_integrations/solaredge.markdown [hunk 1]
```diff
@@ -40,3 +40,15 @@ template:
 ```
 
 {% endraw %}
+
+
+## Energy Dashboard Configuration
+
+As of Home Assistant 2021.11 the SolarEdge intergration can natively populate the [Energy Dashboard](https://www.home-assistant.io/docs/energy/) without additional setup.
```
Comment (frenck) [inline source/_integrations/solaredge.markdown:47]: We don't keep changelogs in our documentation.

File: source/_integrations/solaredge.markdown [hunk 2]
```diff
@@ -40,3 +40,15 @@ template:
...
+
+To monitor your solar system you can utilise `sensor.solaredge_lifetime_energy` for solar production:
```
Comment (frenck) [inline source/_integrations/solaredge.markdown:49]: This is automatically detected, do we need to mention it?

File: source/_integrations/solaredge.markdown [hunk 3]
```diff
@@ -40,3 +40,15 @@ template:
...
+
+If you have a [consumption meter](https://www.solaredge.com/products/metering-and-sensors) in your SolarEdge system you can utilise `sensor.solaredge_imported_energy` and `sensor.solaredge_exported_energy` for energy to and from the grid:
+
+![Screenshot 2021-11-02 21 25 31](https://user-images.githubusercontent.com/79175134/139829667-cfe116f1-c3a7-4c53-8278-6c1d5e7f42d0.png)
```
Comment (frenck) [inline source/_integrations/solaredge.markdown:53]: External screenshot, we cannot use that.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
This PR is running stale, closing it for now. Feel free to re-open when ready to work on it again.

**Pull request closed without merging**